lenses/models/confirmation_task.py

```diff
@@ -337,8 +335,13 @@ class ConfirmationResponse(models.Model):
     confirmation_task = models.ForeignKey(ConfirmationTask, on_delete=models.CASCADE)
     recipient = models.ForeignKey('Users',on_delete=models.CASCADE)
     created_at = models.DateTimeField(auto_now=True)
-    response = models.CharField(max_length=1000, help_text="The response of a given user to a given confirmation task.") 
-    response_comment = models.CharField(max_length=100, help_text="A comment (optional) from the recipient on the given response.") 
+    response = models.CharField(max_length=1000, help_text="The response of a given user to a given confirmation task.",
```

Comment:
The response is predefined in the model definition, so it has to pass code review before it is accepted (usually it is just a 'yes' or 'no'). So, no need to check the language here. We can remove this line. The actual response is 'response_comment' below, that we should check.

Reply:
thanks for the thorough check, will fix these before merge